model-builder/src/builder.rs

```diff
@@ -342,17 +460,21 @@ mod test {
         assert_eq!(
             model.get_corresponding_from_idx(model.vehicle_journeys.get_idx("toto").unwrap()),
             ["A", "B"]
-                .into_iter()
+                .iter()
                 .map(|s| model.stop_points.get_idx(s).unwrap())
                 .collect()
         );
         assert_eq!(
             model.get_corresponding_from_idx(model.vehicle_journeys.get_idx("tata").unwrap()),
             ["C", "D"]
-                .into_iter()
+                .iter()
                 .map(|s| model.stop_points.get_idx(s).unwrap())
                 .collect()
         );
+        let default_calendar = model.calendars.get("default_service").unwrap();
```

Comment:
since it's in a test, I felt I should use the real name to show that it was a calendar like the others. But I'm not really think it helps, so if you prefer I can put the static variable instead

Reply:
Never sure what position to adopt for this kind of case, but I do get your point. I'll let you or others decide. Please resolve when you're satisfied.